app/services/resumeio.py

```
@@ -120,17 +161,30 @@ def __get(self, url: str) -> requests.Response:
+        try:
+            response = requests.get(
+                url,
+                headers={
+                    "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) "
+                    "AppleWebKit/537.36 (KHTML, like Gecko) "
+                    "Chrome/136.0.0.0 Safari/537.36",
+                },
+                timeout=30,  # Add timeout
             )
-        return response
+            
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)